hack/generate-buildah-remote.sh

```diff
@@ -5,8 +5,9 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $SCRIPTDIR/../task-generator
 go build -o /tmp/remote-generator ./remote/main.go
 
-
-/tmp/remote-generator --buildah-task=$SCRIPTDIR/../task/buildah/0.1/buildah.yaml \
-       --remote-task=$SCRIPTDIR/../task/buildah-remote/0.1/buildah-remote.yaml
-/tmp/remote-generator --buildah-task=$SCRIPTDIR/../task/buildah-oci-ta/0.1/buildah-oci-ta.yaml \
-       --remote-task=$SCRIPTDIR/../task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml
+for version in 0.1 0.2; do
+    /tmp/remote-generator --buildah-task=$SCRIPTDIR/../task/buildah/"$version"/buildah.yaml \
+        --remote-task=$SCRIPTDIR/../task/buildah-remote/"$version"/buildah-remote.yaml
+    /tmp/remote-generator --buildah-task=$SCRIPTDIR/../task/buildah-oci-ta/"$version"/buildah-oci-ta.yaml \
+        --remote-task=$SCRIPTDIR/../task/buildah-remote-oci-ta/"$version"/buildah-remote-oci-ta.yaml
```

Comment:
Nitpick, this might be safer, and I like the curlies, they make it a bit more readable:

```suggestion
    /tmp/remote-generator --buildah-task="${SCRIPTDIR}/../task/buildah/${version}"/buildah.yaml \
        --remote-task="${SCRIPTDIR}/../task/buildah-remote/${version}/buildah-remote.yaml"
    /tmp/remote-generator --buildah-task="${SCRIPTDIR}/../task/buildah-oci-ta/${version}/buildah-oci-ta.yaml" \
        --remote-task="${SCRIPTDIR}/../task/buildah-remote-oci-ta/${version}/buildah-remote-oci-ta.yaml"
```

Reply:
Fixed